config/Migrations/20251016065152_AddUsersLastContributionColumn.php

```diff
@@ -0,0 +1,23 @@
+<?php
+use Migrations\AbstractMigration;
+
+class AddUsersLastContributionColumn extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     * @return void
+     */
+    public function change()
+    {
+        $table = $this->table('users');
+        $table->addColumn('last_contribution', 'integer', [
```

Comment:
That’s really not a big deal, but the position of the `last_contribution` column is different here than what you specified in the fixtures. The migration script adds the column as the last position, unless specified with the option `after`, e.g. `'after' => 'last_time_active'`.

You can use `cake migrations rollback` and `cake migrations migrate` to re-run the migration, and check how the table looks with `desc users;` in the mysql shell.